karma.conf.js

```diff
@@ -70,18 +77,37 @@ module.exports = (config) => {
                     lines: 80,
                     excludes: ['src/plugins/plot/**/*.js']
                 }
-            }
+            },
+            reporters: [
+                'spec',
+                'coverage-istanbul',
+                'html'
+            ]
         },
         // HTML test reporting.
         htmlReporter: {
             outputDir: "dist/reports/tests",
             preserveDescribeNesting: true,
             foldAll: false
         },
+        coverageIstanbulReporter: {
+            fixWebpackSourcePaths: true,
+            dir: process.env.CIRCLE_ARTIFACTS ?
+                process.env.CIRCLE_ARTIFACTS + '/coverage' :
+                "dist/reports/coverage",
+            reports: ['html', 'lcovonly', 'text-summary'],
+            thresholds: {
+                global: {
+                    statements: 50,
```

Comment:
Let's keep the `lines` threshold, and remove the others for now. Let's set it to 62.3 (the current level of coverage).

Reply:
sure, np

Reply:
updated.